app/server/src/types.js

```diff
@@ -82,6 +84,7 @@ type SanitizedValues = {
 
 type Generator = {
   profileSection: (basics?: Basics) => string,
+  aboutSection: (basics?: Basics, heading?: string) => string,
```

Comment:
Until the rest of the templates also have an `aboutSection` method, can we mark this method as optional to prevent type errors?

```suggestion
  aboutSection?: (basics?: Basics, heading?: string) => string,
```